scripts/network-init

```diff
@@ -108,21 +108,14 @@ rename_network_label() {
     done
 }
 
-maybe-enable-tls-verification() {
-  verification_enabled="$(${XE} pool-list params=tls-verification-enabled --minimal)"
-  if [ "$FIRSTBOOT_ENABLE_TLS_VERIFICATION" == "true" ] && [ "$verification_enabled" != "true" ] ; then
-      ${XE} pool-enable-tls-verification
-  fi
-}
-
 if [ "$UPGRADE" = "true" ]; then
     rename_network_label
 else
     prepare_networking
     rename_network_label
     # Ensure changes are synced to disk
     xe pool-sync-database
```

Comment:
I wonder why plain xe is used here if `$XE `is defined, It would make sense to use only one of them.